source/main.bmx

```diff
@@ -5986,13 +5992,14 @@ endrem
 			GetProgrammeProducerCollection().UpdateAll()
 		EndIf
 
-		If minute = 37
+		If TSaveGame.autoSaveNow and Not GetPlayer().GetFigure().IsInRoom()
 			Local gameName:String = GameConfig.savegame_lastUsedName
 			Local autoSaveName:String = "autosave.xml"
-			If gameName and gameName <> ""
+			If gameName and gameName <> "" and gameName <> "quicksave"
```

Comment:
`if gameName` und `If gameName <> ""` fuehren in BlitzMax zum Gleichen Ergebnis.